src/main/java/at/ac/tuwien/kr/alpha/common/atoms/Atom.java

```diff
@@ -132,4 +133,13 @@ default int compareTo(Atom o) {
 		return 0;
 	}
 
+	/**
+	 * Returns whether this Atom is a builtin-atom according to the ASP standard. This covers the atoms currently defined by <code>ComparisonAtom</code>.
+	 * 
+	 * @return true if this Atom is a comparison atom, false otherwise.
```

Comment:
This comment does not really describe the code below (it describes the intent of the method though, but that should rather be part of the description above).